test/snippet/search/configuration_modes.cpp

```diff
@@ -3,16 +3,11 @@
 int main()
 {
     // Report the hit with the least number of errors (either 0 or 1 errors).
-    seqan3::configuration const cfg1 = seqan3::search_cfg::max_error{seqan3::search_cfg::total{1},
-                                                                     seqan3::search_cfg::substitution{0},
-                                                                     seqan3::search_cfg::insertion{1},
-                                                                     seqan3::search_cfg::deletion{1}} |
-                                       seqan3::search_cfg::mode{seqan3::search_cfg::best};
+    seqan3::configuration const cfg1 = seqan3::search_cfg::max_error{seqan3::search_cfg::total{1}}
+                                     | seqan3::search_cfg::reporting_best_hit;
 
     // Report all hits with best + 1 error. E.g., if the best hit has 1 error, all hits with 1 and 2 error are reported.
-    seqan3::configuration const cfg2 = seqan3::search_cfg::max_error{seqan3::search_cfg::substitution{0},
-                                                                     seqan3::search_cfg::insertion{1},
-                                                                     seqan3::search_cfg::deletion{1}} |
-                                       seqan3::search_cfg::mode{seqan3::search_cfg::strata{1}};
+    seqan3::configuration const cfg2 = seqan3::search_cfg::max_error{seqan3::search_cfg::total{1}}
+                                     | seqan3::search_cfg::reporting_strata_hits{1};
```

Comment:
Oh yes... But I think we agreed that we put the `|` at the start of each new line and below the `=` sign?

Reply:
Well, only for the views 💥, because ranges library does it. Before that, the rule was the line ends with the operator and the expression continues on the new line aligned with the previous expression. Now, of course we have exactly these inconsistencies beacuse the reasoning is just absurd 🔫 . I don't want to extend this to a lengthy discussion, but since it would be weird to suddently change this behaviour I am in favor of keeping the old rule. So only pipes of views are wrapped to the next line. All others get to the end of line.
See here: https://github.com/seqan/seqan3/wiki/General-formatting#operators-and-parentheses

Reply:
Alright

Reply:
but it is not resolved